dkms.in

```diff
@@ -1061,6 +1061,12 @@ prepare_build()
             $"Check $build_dir for more information."
     done
 
+    if [ -f "$kernel_source_dir/.kernelvariables" ]; then
+        export CC=$(echo -e "show-%:\n\t@echo \$(\$*)\ninclude $kernel_source_dir/.kernelvariables" | make -f - show-CC)
+    else
+        unset CC
+    fi
+
     if [[ -e "${kernel_config}" ]]; then
```

Comment:
Out of curiosity: how did the issue manifest itself?

Eg. gcc-10 was supposed to be used, while gcc (which links to gcc-x) ended in use. ... etc